internal/server/route.go

```diff
@@ -82,6 +82,8 @@ func (s *Server) initLeasingHistoryRoutes() {
 	historyRoutes.Post("/review/:id", s.handler.leasingHistory.CreateReview)
 	historyRoutes.Patch("/review/:id", s.handler.leasingHistory.UpdateReview)
 	historyRoutes.Delete("/review/:id", s.handler.leasingHistory.DeleteReview)
+	historyRoutes.Post("review/:id/image", s.handler.leasingHistory.UploadReviewImage)
+	historyRoutes.Delete("review/image/:url", s.handler.leasingHistory.DeleteReviewImageByURL)
```

Comment:
I forgot to add the `/` before the review. Please change all routes to `/:id/review`. I apologize for my mistake in the review.